mbed_host_tests/host_tests_conn_proxy/conn_proxy.py

```diff
@@ -28,7 +28,7 @@
 class KiViBufferWalker():
     """! Simple auxiliary class used to walk through a buffer and search for KV tokens """
     def __init__(self):
-        self.KIVI_REGEX = r"\{\{([\w\d_-]+);([^\}]+)\}\}\n"
```

Comment:
@mazimkhan Could you clarify a bit? I'm not sure what you mean. Are you saying you want to completely remove `\n` and/or `\r` from the KV regex?


The `\n` had been there before, but at when point @PrzemekWirkus had removed it, which caused issues in DAPLink, so I just added it back in that PR.